src/main/java/org/broadinstitute/hellbender/tools/exome/eval/ConvertGSVariantsToSegments.java

```diff
@@ -33,15 +34,51 @@
 import java.util.stream.Stream;
 
 /**
- * Tool to convert Genotype Strip Variant call file into {@link HiddenStateSegmentRecord} table file.
+ * Converts Genome STRiP (GS) variant call format (VCF) variants into {@link HiddenStateSegmentRecord} CopyNumberTriState format calls.
```

Comment:
No. I don't git add these extra files so it's not clear to me why they appear in my commit.